### PR TITLE
Adds Cash Registers to cargo order list

### PR DESCRIPTION
**Pull request by Fruerlund**
You can now order a CASH register also known as Point of Sale(POS) from CARGO. It's basicly a functioning cash register. You can add items to your list, and people can pay with cash or card. The register also keeps till so make sure noone steals your credits.

This will allow people to set up small shops and to encourage roleplay and the use of economy.

🆑Fruerlund
tweak: Updates the POS system. Adds a few new procs and changes the unit to credits Cr
add: You can now order it via CARGO.
/🆑

### CODE REVIEW

File: code/modules/economy/POS.dm [hunk 1]
```diff
@@ -124,7 +123,7 @@ var/const/POS_HEADER = {"<html>
 	icon_state = "pos"
 	density = 0
 	name = "point of sale"
-	desc = "Also known as a cash register, or, more commonly, \"robbery magnet\"."
+	desc = "Also known as a cash register, or, more commonly, \"robbery magnet\" Needs to be ancored for it to be used."
```
Comment (FlattestGuitar) [inline code/modules/economy/POS.dm]: anchored

Comment (Fruerlund) [inline code/modules/economy/POS.dm]: woopsie.

**Comment by taukausanake** [comment]
This is a good idea, because economy, but it's going to be a pain to actually implement even on a per-player basis. I want to say historically no economy in SS13 has ever worked. I really want one to work too

**Comment by BiberDark** [comment]
Why not make all people start with 0 Credits and the heads have to authorize all payments after every 20-30 minutes and on for each person. So only working people get money.

**Comment by Fruerlund** [comment]
@BiberDark not in the scope in this PR, but a basic pay check system in the future would be an interesting idea.

**Comment by granodd** [comment]
Economy systems don't work on SS13 simply because money doesn't carry over from shift to shift. You don't see those people who won a million credits on the slot machine still having those around.

Money doesn't mean anything because it resets every shift, and therefore has little impact or purpose outside of purely RP element.

That being said, cash registers would be neat to have anyways for people who want to make a little shop or whatnot.

**Comment by BiberDark** [comment]
Then why not add a cash , maybe saved like a the karma points. And as money think you have to pay for your equipment each shift, and food.

**Comment by Fruerlund** [comment]
I'd like folks to remember, that this PR is not mainly to increase the use of economy. It's also a tool for rolepaly purposes for those people who like to charge credits for their stuff / booze etc. etc.

In the future perhaps a pay check system can be added and all stuff coming from vendors should be priced. Clothes, tools etc. That would at least be a big step.

**Comment by ModDownloading** [comment]
I saw someone that opened up an actual restaurant with one of these, by converting the kitchen into more seating space and putting the kitchen stuff in the garden area behind. It worked surprisingly well, as people were willing to stand in lines and get interesting food rather than just run to the vending machines (usually). 

I saw it in action mainly because this was the round involving an election for a command position and I was airing my views staunchly opposed to the democratic process (which, predictably, led to much debate, which I wanted anyways) over the radio whilst standing in the restaurant.

**Comment by Fox-McCloud** [comment]
We already have the EFTPOS system and scanners literally everywhere and literally no one uses them *ever*.

I don't see why this would be any different.

**Comment by taukausanake** [comment]
EFTPOS is ugly though. I wouldn't use it. Rather use a cash system. Register can use credits on ID but the EFTPOS I rather not

**Comment by Fruerlund** [comment]
@Fox-McCloud 

The cash register is far easier to use for both the customer and the cashier, as you don't have to hand it over to the other player.

It also takes space cash which hopefully will make players use it just a bit more.

This PR besides adding the ability to hold the till in the cash also fixes the fact, that the cash regsiter prior to this PR gives back the proper change but also /spawns/ a new space cash with the amount you gave so it duplicated cash

**Comment by TullyBurnalot** [comment]
First off, allow us to extend our apologies for the PR sitting idle for so long. Unfortunately, it was decided not to merge it; the economy system, as is, is poorly implemented and borderline useless, with elements of it strewn about and rarely, if ever, used. The currently available means of accessing and distributing currency are completely functional, and yet rarely see use; adding cash registers to the mix would border on redundancy.

As daunting as it is, it would be far preferrable to FIRST make the economy system meaningful in and of itself, and then make additions to it, rather than the other way around.

**Pull request closed without merging**